staging/src/k8s.io/pod-security-admission/policy/check_procMount.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/pod-security-admission/api"
+)
+
+func init() {
+	addCheck(CheckProcMount)
+}
+
+// CheckProcMount returns a baseline level check that restricts
+// setting the value of securityContext.procMount to DefaultProcMount
+// in 1.0+
+func CheckProcMount() Check {
+	return Check{
+		ID:    "procMount",
+		Level: api.LevelBaseline,
+		Versions: []VersionedCheck{
+			{
+				MinimumVersion: api.MajorMinorVersion(1, 0),
```

Comment:
note to self: the procMount field was added in 1.12+, but since this is forbidding an escalating value in a new field, it's ok for this to be included in the 1.0+ policy levels (it would still pass a pod that omitted the value in a server prior to 1.12)

Reply:
ah, the integration tests are unhappy because this is still an alpha field, so it is not persisted unless the ProcMountType feature gate is enabled... the check implementation looks correct (forbid an escalating value in the field), but the failure cases are tricky to write if the feature that allows data to be persisted in that field is disabled... let me think about how to test that

Reply:
opened https://github.com/kubernetes/kubernetes/pull/103365 to let the failure cases declare features they depend on

use for this would look like the last commit of https://github.com/liggitt/kubernetes/commits/proc-mount-baseline

Reply:
https://github.com/kubernetes/kubernetes/pull/103365 is merged, can rebase on master and set `failRequiresFeatures` like https://github.com/liggitt/kubernetes/commit/0caca308702055eb48eda7c58bfbaf32c0ccc72e and this should be good

Reply:
done
thanks @liggitt :)